### PR TITLE
Add cursorless settings command

### PR DESCRIPTION
**Pull request by pokey**
It's a bit awkward to access Cursorless VSCode settings by voice today.  I was hoping to show how to do so in the new tutorial, but would be nice if it were an easier way to show them

## Checklist

- [ ] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)

### CODE REVIEW

File: cursorless-talon/src/cursorless.talon [hunk 1]
```diff
@@ -24,3 +24,8 @@ cursorless instructions:   user.cursorless_open_instructions()
 cursorless docks:          user.cursorless_open_instructions()
 cursorless reference:      user.cursorless_cheat_sheet_show_html()
 cursorless cheat sheet:    user.cursorless_cheat_sheet_show_html()
+cursorless settings:
```
Comment (pokey) [inline cursorless-talon/src/cursorless.talon:27]: One concern is that we also have Talon-side configuration, eg spoken forms, so I'm slightly hesitant for this voice command to mean VSCode settings

File: cursorless-talon/src/cursorless.talon [hunk 2]
```diff
@@ -24,3 +24,8 @@ cursorless instructions:   user.cursorless_open_instructions()
...
+    sleep(50ms)
+    user.vscode("workbench.action.openGlobalSettings")
+    sleep(250ms)
```
Comment (pokey) [inline cursorless-talon/src/cursorless.talon:30]: Not in love with this and the other arbitrary sleep 🤔

Comment (AndreasArvidsson) [reply cursorless-talon/src/cursorless.talon:30]: We already have those for other commands so I guess it's fine until we have something better, but why the leading sleep?

File: cursorless-talon/src/cursorless.talon (hunk 1 above)
Comment (AndreasArvidsson) [reply cursorless-talon/src/cursorless.talon:27]: We could of course open a little imgui with buttons/voice commands that will open a csv file or vscode settings

File: cursorless-talon/src/cursorless.talon (hunk 2 above)
Comment (pokey) [reply cursorless-talon/src/cursorless.talon:30]: Yeah good question.  I have it in my personal "show settings" command.  Presuming I added it for a reason 🤷

File: cursorless-talon/src/cursorless.talon (hunk 1 above)
Comment (pokey) [reply cursorless-talon/src/cursorless.talon:27]: Yeah that's fair; we could always do that later, so prob fine to ship it like this for now

File: cursorless-talon/src/cursorless.talon (hunk 2 above)
Comment (AndreasArvidsson) [reply cursorless-talon/src/cursorless.talon:30]: Feels like a left over from something else maybe? I would remove it.

**Review by AndreasArvidsson — APPROVED** [review]

**Pull request merged**